mozregression/json_pushes.py

```diff
@@ -45,18 +41,18 @@ def _request(self, url, check_changeset=None):
             )
         return pushlog
 
-    def pushlog_for_change(self, changeset):
+    def pushlog_for_change(self, changeset, **kwargs):
         """
         Returns the json pushlog object that match the given changeset.
 
         A MozRegressionError is thrown if None is found.
         """
         return next(self._request(
-            self.json_pushes_url(changeset=changeset),
+            self.json_pushes_url(changeset=changeset, **kwargs),
```

Comment:
You might be able to save the call to json_pushes_url by using request's builtin support for taking parameters. We do this in the treeherder client: 

https://github.com/mozilla/treeherder/blob/master/treeherder/client/thclient/client.py#L686